plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1375,6 +1375,25 @@ public void testCheckpointing()
         assertThat(listCheckpointFiles(transactionLogDirectory)).hasSize(2);
         assertQuery("SELECT * FROM " + tableName + " WHERE a_string <> 'fill'", "VALUES (1,'ala'),  (3,'psa'), (2, 'bobra')");
 
+        // replace table
```

Comment:
you can do as well the same for a `CREATE OR REPLACE` statement (without `AS SELECT ...`)